### DISCUSSION TITLE
Development stage

### DISCUSSION THREAD

**Discussion by ptabandzelic** [databrickslabs/dbx#470] [Q&A]
Hey, would u recommend using databricks connect  + dbx for development stage even tho connect is not being maintained anymore.
Is the basic worfkflow for dev stage:
IDE -> dbx execute or databricks connect to run tasks on all purpose clusters
Databricsk REPO UI directly 
Run on interactive clusters only
How would u go about just running a jupyter notebook on databricks cluster without needing to create workflow and task for it?

**Comment by renardeinside** [comment]
> How would u go about just running a jupyter notebook on databricks cluster without needing to create workflow and task for it?

I wouldn't recommend using Jupyter notebooks - Databricks Notebooks provide way richer interface, proper versioning and no need for local setup. 

Ways to combine IDE and Databricks Notebooks are described here - https://dbx.readthedocs.io/en/latest/guides/python/devloop/mixed/

Everything else is correct.
